torchaudio/datasets/ljspeech.py

```diff
@@ -63,7 +63,7 @@ def __init__(self,
                     download_url(url, root, hash_value=checksum)
                 extract_archive(archive)
 
-        with open(self._metadata_path, "r") as metadata:
+        with open(self._metadata_path, "r", newline='') as metadata:
```

Comment:
I assume this change is required for Windows. Is that correct?

Reply:
Yes, there were issues with the Windows binary checks that are run, and it appears to be a newline issue. This solution fixes the problem, and is the recommended way of opening a CSV file according to Python docs.